pom.xml

```diff
@@ -828,6 +819,62 @@
       </build>
     </profile>
 
+    <profile>
```

Comment:
An alternative to this Maven profile would have been moving `spotless:check` as the first goal [here](https://github.com/dylibso/chicory/blob/4a00969cc4aa51a3532f860df40083319b91b271/.github/workflows/ci.yaml#L50) ?